installation/windows.md

```diff
@@ -5,16 +5,67 @@ title: Windows
 
 {% include base.html %}
 
-# Installation on Windows
+# openHAB on Windows
 
-The openHAB runtime comes as a platform-independent zip file.
-To install it, follow these simple steps.
-You will need to install Java if not already installed. Go to http://java.com to get it.
-Unzip the `openhab-offline-<version>.zip` to where it is intended to be running from, e.g. `C:\openhab`.
-Note that there can't be a space in the path, so it can't be in "Program Files" for example.
+The following instructions will guide you through the process of setting up openHAB 2.
 
-Start the server: Launch the runtime by executing the script `start.bat`.
+## Prerequisites
 
-Point your browser to `http://localhost:8080` and you should be looking at the openHAB portal page.
+OpenHAB requires Java 8 to run correctly, we reccomend a version of 8_101 and higher. 
+To get the latest version, visit the [java website](https://www.java.com/en/download/).
+
+The directory that openHAB is installed to **contains no spaces**, on this page, we will be using `C:\openhab` as an example.
+
+## Installation
+
+The openHAB runtime is distributed using a platform-independent zip file. To install it, follow these simple steps:
+
+1. Choose between the [RC (Release client) version](https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab%2F2.0.0.RC1%2Fopenhab-2.0.0.RC1.zip) or the [latest snapshot version](https://openhab.ci.cloudbees.com/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab/target/openhab-2.0.0-SNAPSHOT.zip) of openHAB2.
+
+2. Unzip the file in your chosen directory (e.g. `C:\openhab`)
+
+3. Start the server: Launch the runtime by executing the script `start.bat`.
+
+4. Point your browser to `http://localhost:8080` and you should be looking at the openHAB portal page. 
+  On this website, you can setup many of the [things]({{base}}/configuration/things.html) and [items]({{base}}/configuration/items.html) for your smart home.
```

Comment:
That's not really what you can do there. In fact the links you've mentioned mainly describe how to do definitions in files.

Maybe point to the UIs overview page here? http://docs.openhab.org/addons/uis.html

Before that I believe the Initial Setup Screen will be shown. http://docs.openhab.org/configuration/packages.html